tests/models/test_cli.py

```diff
@@ -127,6 +127,7 @@ def test_model_with_no_deployable_flavors_fails_pollitely():
         assert "No suitable flavor backend was found for the model." in stderr
 
 
+@pytest.mark.large
```

Comment:
Moved some tests to large -- reduces test time by about 3 minutes (~9 to ~6). Actually, `test_expected_tags_logged_when_using_conda` didn't work in this runner -- wanted to avoid debugging that for now.